services/dynamic-sidecar/tests/unit/test_core_rabbitmq.py

```diff
@@ -0,0 +1,183 @@
+# pylint: disable=unused-argument
+# pylint: disable=redefined-outer-name
+import asyncio
+import json
+import os
+import uuid
+from asyncio import AbstractEventLoop
+from pathlib import Path
+from pprint import pformat
+from typing import Iterator, List
+from unittest import mock
+
+import aio_pika
+import pytest
+from _pytest.fixtures import FixtureRequest
+from fastapi.applications import FastAPI
+from models_library.projects import ProjectID
+from models_library.projects_nodes import NodeID
+from models_library.settings.rabbit import RabbitConfig
+from models_library.users import UserID
+from pytest_mock.plugin import MockerFixture
+from simcore_service_dynamic_sidecar.core.rabbitmq import SLEEP_BETWEEN_SENDS, RabbitMQ
+from simcore_service_dynamic_sidecar.modules import mounted_fs
+
+pytestmark = pytest.mark.asyncio
+
+
+pytest_plugins = [
+    "pytest_simcore.docker_compose",
+    "pytest_simcore.docker_swarm",
+    "pytest_simcore.rabbit_service",
+    "pytest_simcore.repository_paths",
+    "pytest_simcore.tmp_path_extra",
+]
+
+pytest_simcore_core_services_selection = ["rabbit"]
+
+# FIXTURE
+
+
+@pytest.yield_fixture(scope="module")
```

Comment:
what does this yield_fixture do?

Reply:
an autocomplete error, replaced with fixture